### ISSUE TITLE
Ouvrir l'inscription aux utilisateurs

### ISSUE THREAD

**Issue by remikel** [awenig/datan#125]
Ca permettrait d'avoir plus de votes décryptés et puis tu pourrais toujours les modérer. Si tu vois un utilisateur qui fait plusieurs décryptages bien objectif et tout, tu peux le passer en modérateur.
Il faudrait juste rajouter des rôles quoi.

**Comment by awenig** [comment]
Good idea. Il faudra alors créer une page qui permettrait de "gérer son compte", du style changer le MDP, adresse e-mail, supprimer le compte, etc. Et puis mettre tout ceci à jour au niveau RGPD

**Comment by IGLOU-EU** [comment]
S'inspirer du fonctionnement des wikis peut être une bonne piste.
Pour éviter tous problèmes rgpd, le mieux est encore de n'utiliser aucunes informations personnelles, et anonymiser les emails en cas de suppression de compte.
Il est aussi possible de ne pas avoir de mot de passe à gérer en utilisant des "MagicLinks"

**Comment by awenig** [comment]
Merci. Pour le moment, c'est un projet une issue en stand-by car il n'y a pas un intérêt pour un utilisateur (autre que député, membre de l'équipe), d'avoir un compte. Mais à voir comment le site évolue dans le futur.
